src/panels/config/integrations/integration-panels/bluetooth/bluetooth-advertisement-monitor.ts

```diff
@@ -167,6 +180,7 @@ export class BluetoothAdvertisementMonitorPanel extends LitElement {
           scanner?.name ||
           row.source,
         device: device?.name_by_user || device?.name || undefined,
+        datetime: new Date(row.time * 1000),
```

Comment:
`time` is only updated when the data changes. Even if we see it again and the data is the same it won't trigger a new callback.

It might be more accurate to call this `Last updated` or `Last changed`

Reply:
Got it, in my system I have a bunch of rows updating every second, so I assumed it was reporting every advertisement, but I guess those noisy advertisements are also sending seemingly random data each time as well. 

> Last updated or Last changed or maybe Age

Have a preference? I know this is kind of your thing, hope I didn't overstep. I'm also open to having it hidden by default if you don't think it's as useful as the other fields.

Reply:
> Have a preference? I know this is kind of your thing, hope I didn't overstep. 

Not at all. We are all working on this together. Happy to see someone else working on it as well.

> I'm also open to having it hidden by default if you don't think it's as useful as the other fields.

Thinking about it some more, I think I'd go with simply `Updated`

Reply:
https://github.com/home-assistant/frontend/pull/24439